README.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
💄 I think this is leftover from my other PR, which was done through GitHub UI so it didn't get formatted correctly until now.